WebContent/WOH-addRestaurant.jsp

```diff
@@ -53,20 +53,20 @@
 						<div class="form-group">
 						<label class="control-label col-sm-2" for="WOH-addRestaurant-distanceInput">Distance</label>
 							<div class="col-sm-10">
-	         					<input type = "text" class="form-control WOH-addRestaurant-distanceInput" name = "distance">
+	         					<input type = "text" class="form-control WOH-addRestaurant-distanceInput" value = 0 name = "distance">
 	         				</div>
 	         			</div>
 	         			
 	         		<!-- Food Type Dropdown -->
 						<div class="form-group">
 							<div class="col-sm-2">
-		  						<label class="control-label" for="WOH-addRestaurant-typeInput">Select Food Type</label>
+		  						<label class="control-label" for="WOH-addRestaurants-typeInput">Select Food Type</label>
 		  					</div>
 		  					<div class="col-sm-10">
 		  						<select class="form-control WOH-addRestaurants-typeInput" name="type">
-		    						<option value = "fastFood" >Fast Food</option>
-		    						<option value = "sitDown" >Sit-Down</option>
-		    						<option value = "dessert" >Dessert</option>
+		    						<option selected="selected" value = "1" >Fast Food</option>
```

Comment:
This is good for now, but in the future, I want you to set a placeholder for all of the drop downs.
<option value="" disabled selected>Select your option</option>
you could do something like this.